conf.d/http_check.yaml.example

```diff
@@ -77,6 +77,11 @@ instances:
     # The SSL certificate will always be validated for this additional
     # service check regardless of the value of disable_ssl_validation
     #
```

Comment:
Yeah, to me as well but it seems that it wasn't for everybody... I'll delete that and we'll reconsider that decision if support gets more tickets about that part of the check ;-)